targets/mbedos5/source/jerry_port_mbed.c

```diff
@@ -90,10 +90,36 @@ jerry_port_get_time_zone (jerry_time_zone_t *tz_p) /**< timezone pointer */
 /**
  * Implementation of jerry_port_get_current_time.
  *
- * @return current timer's counter value in microseconds
+ * @return current timer's counter value in milliseconds
  */
 double
 jerry_port_get_current_time ()
 {
-  return (double) us_ticker_read ();
+  static uint32_t last_tick = 0;
+  static time_t last_time = 0;
+  static uint32_t skew = 0;
```

Comment:
I think the only thing which we might consider is that the threads. If the JerryScript can execute JS in parallel threads safely, this function should be reviewed again. I can imagine that some kind of lock will be done around the port functions in this case no to rely the port itself.
On the other hand, static variables in functions can be seen in all over the ports (even in mbed).
What kind of improvements do you see about this static variables not to be so confusing?